Description: revert the upstream deprecation of the which command
 Original commit: https://salsa.debian.org/debian/debianutils/-/commit/3a8dd10b4502f7bae8fc6973c13ce23fc9da7efb
Origin: upstream
Forwarded: not-needed
Last-Update: 2021-11-30

--- a/which
+++ b/which
@@ -1,8 +1,6 @@
 #! /bin/sh
 set -ef
 
-echo >&2 "$0: this version of \`which' is deprecated; use \`command -v' in scripts instead."
-
 if test -n "$KSH_VERSION"; then
 	puts() {
 		print -r -- "$*"
--- a/which.1
+++ b/which.1
@@ -1,5 +1,5 @@
 .\" -*- nroff -*-
-.TH WHICH 1 "9 Jul 2021" "Debian"
+.TH WHICH 1 "29 Jun 2016" "Debian"
 .SH NAME
 which \- locate a command
 .SH SYNOPSIS
@@ -27,11 +27,3 @@
 .TP
 .B 2
 if an invalid option is specified
-.SH DEPRECATION
-.IP
-Since
-.B type
-and
-.B command \-v
-were mandated by POSIX, this utility is no longer useful for
-maintainer scripts and thus will be removed from debianutils.
